.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,18 @@
+name: Semgrep
+on:
+  pull_request: {}
+  push:
+    branches: ["master"]
+jobs:
+  semgrep:
+    name: Scan
+    runs-on: [self-hosted]
```

Comment:
@ShankarParimi its an open source repo, it wont run on self-hosted nodes, use `runs-on: ubuntu-latest`

Reply:
made the change.